### DISCUSSION TITLE
DietPi-backup changing the time in cron

### DISCUSSION THREAD

**Discussion by jalsco** [MichaIng/DietPi#5951] [Q&A]
Can you please confirm about this - 

My understanding is - if I set `dietpi-backup` as "DAILY" in the UI - then it is included in the `dietpi` cron script.  I do not know exactly what time that is set to run....

But if I  DO NOT want to change the behavior of the whole  `dietpi` script in cron - because there seems like a lot else in there, not just `dietpi-backup`.... But I DO want to be specific about exactly "when" ( time and other frequency settings ) that  `dietpi-backup`  runs...  So that means I should set "DAILYBACKUPS" to NO in the `dietpi-backup` UI ... and then just create my own `cron` entries ..   

Seems like no other way to set the TIME that backup runs.  Is it true ?  either (1) change time of the whole `dietpi` job OR (2)  own cron job(s) to run `dietpi-backup` with "NO" to daily in the UI..... 

Did I understand it right ?

**Comment by MichaIng** [comment]
Yes, `dietpi-backup` uses the daily cron job for daily backup. You can see the execution code in `/etc/cron.daily/dietpi`.

You can see and adjust the daily cron job execution time via `dietpi-cron`. This is safe to do, nothing is expecting or requiring a specific time for any of those hourly/daily/weekly/monthly cron jobs. The only measure is that you wouldn't want it to disturb you when you're working on the system, so you'd want the daily job to run while you are usually at sleep 🙂. And to not run everything the same minute, you'd want to start hourly, daily, weekly and monthly cron jobs with a few minutes between them. But even all at once does not generally cause issues.

Of course you can also run `/boot/dietpi/dietpi-backup 1` with an own cron job, but I see no reason why one would do that.

**Reply by jalsco** [reply]
i did not realize there was some ui menus `dietpi-cron`  
Now knowing that, I will let the dietpi system manage all that... But probably - also - I will take a `dd` image backup of the boot partition in my own `cron` job - probably only weekly. .. thanks again for your responsiveness.

**Reply by MichaIng** [reply]
> i did not realize there was some ui menus `dietpi-cron`

Still something to discover 😃. It does btw nothing else than adjusting `/etc/crontab`, which is the responsible for these `/etc/cron.*` script drop directories on Linux systems in general.

Indeed, a full `dd` disk backup is a good thing by times, in case the system doesn't boot anymore at all.

**Reply by Joulinar** [reply]
an overview of our tools is available at https://dietpi.com/docs/dietpi_tools/
